File: output/Llama-2-100/avatar/Java/Python/codeforces_99_A.py
def codeforces_99_A ( str ):
    dotIndex = 0
    for i in range ( len ( str ) ):
        if str [ i ] == '.':
            dotIndex = i
            break
        if str [ dotIndex - 1 ] == '9':
            print ( "GOTO Vasilisa." )
        else:
            _str = ''
            for j in range ( dotIndex ):
                _str += str [ j ]
            print ( int ( _str ) + 1 )

File: output/Llama-2-100/avatar/Python/Java/codeforces_99_A.java
import java.util.Scanner;

public class codeforces_99_A {

    public static void main ( String [] args ) {

        Scanner sc = new Scanner ( System.in );
        String l = sc.nextLine ( );
        String [] split = l.split ( "." );
        String s = String.join ( "", split [ 0 ] );
        int i = Integer.parseInt ( split [ 1 ] );

        if ( s.charAt ( s.length ( ) - 1 ) == '9' ) {
            System.out.println ( "GOTO Vasilisa." );
        } else if ( s.charAt ( s.length ( ) - 1 ) != '9' && i < 5 ) {
            System.out.println ( s );
        } else {
            s = String.join ( "", s );
            System.out.println ( Integer.parseInt ( s ) + 1 );
        }

    }

}
